src/_locales/ar/messages.json

```
@@ -0,0 +1,429 @@
+      "message": "عدد وحدات الحدود حول رمز الاستجابة السريعة الملونة في لون الخلفية",
+      "description": "The description of the quiet zone option."
+    },
+    "optionQrQuietZoneDescrLink": {
+      "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ#what-is-a-quiet-zone",
+      "description": "The 'Learn more' link in the description of the quiet zone option."
+    },
+  
+    "optionQrCodeColor": {
+      "message": "لون رمز الاستجابة السريعة:",
+      "description": "This is an option shown in the add-on settings."
+    },
+  
+    "optionQrCodeBackgroundColor": {
+      "message": "لون خلفية رمز الاستجابة السريعة:",
+      "description": "This is an option shown in the add-on settings."
+    },
+  
+    "optionErrorCorrection": {
+      "message": "مستوى تصحيح الخطأ:",
+      "description": "Error correction setting for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+    },
+    "optionEcLow": {
+      "message": "(7%) منخفض",
+      "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+    },
+    "optionEcMedium": {
+      "message": "(15%) متوسط",
+      "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+    },
+    "optionEcQuartile": {
+      "message": "(25%) ربع",
+      "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+    },
+    "optionEcHigh": {
+      "message": "(30%) مرتفع",
+      "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+    },
+    "optionErrorCorrectionDescr": {
+      "message": "تشير هذه القيمة إلى مدى فقد الرمز ، بحيث لا يزال من الممكن مسحه ضوئيًا",
+      "description": "The description of the error correction option."
+    },
+    "optionErrorCorrectionDescrLink": {
+      "message": "https://en.wikipedia.org/wiki/QR_code#Error_correction",
+      "description": "The 'Learn more' link in the description of the error correction option."
+    },
+  
+    "optionAutoGetSelectedText": {
+      "message": "استخدم النص المحدد على موقع الويب تلقائيًا",
+      "description": "This is an option shown in the add-on settings."
+    },
+    "optionUseMonospaceFont": {
+      "message": "استخدام الخط monospace",
+      "description": "This is an option shown in the add-on settings."
+    },
+  
+    "optionDebugMode": {
+      "message": "تمكين وضع التفحيص",
+      "description": "This is an option shown in the add-on settings."
+    },
+    "optionDebugModeDescr": {
+      "message": "هذا مفيد فقط للحصول على ناتج تسجيل أكثر تفصيلاً في وحدة التحكم للإبلاغ عن الأخطاء ، إلخ",
+      "description": "The description shown for the debug mode (optionDebugMode)."
+    },
+  
+    "translatorCredit": {
+      "message": "This add-on has been translated into Arabic by Ahmad Ayman $LINK$ !",
```

```suggestion
      "message": "This add-on has been translated into Arabic by Ahmad Ayman ($LINK$)!",
```

* remove space (it is not needed is it?)
* only link part inside of brackets

(see [doc](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on) for reference)